packages/flutter_tools/lib/src/ios/ios_emulators.dart

```diff
@@ -39,22 +50,29 @@ class IOSEmulator extends Emulator {
 
   @override
   Future<void> launch() async {
-    Future<bool> launchSimulator(List<String> additionalArgs) async {
-      final List<String> args = <String>[
-        'open',
-        ...additionalArgs,
-        '-a',
-        globals.xcode.getSimulatorPath(),
-      ];
-
-      final RunResult launchResult = await globals.processUtils.run(args);
-      if (launchResult.exitCode != 0) {
-        globals.printError('$launchResult');
-        return false;
-      }
-      return true;
-    }
+    await launchSimulator(<String>[]);
+    return _simulator.boot();
```

Comment:
We don't need to run `open` a second time since the next thing that happens is booting the specified simulator.